NeoML/test/src/DnnBlobTest.cpp

```diff
@@ -25,10 +25,12 @@ using namespace NeoMLTest;
 
 TEST( CDnnBlobTest, InitWindowBlob )
 {
-    CPtr<CDnnBlob> parent = CDnnBlob::CreateDataBlob( MathEngine(), CT_Float, 16, 1, 1 );
-    CPtr<CDnnBlob> blob = CDnnBlob::CreateWindowBlob( parent );
+    CPtr<CDnnBlob> parent = CDnnBlob::CreateDataBlob(MathEngine(), CT_Float, 16, 1, 1);
+    CPtr<CDnnBlob> window = CDnnBlob::CreateWindowBlob(parent, 16);
 
-    EXPECT_FALSE( blob->GetData().IsNull() );
+    EXPECT_TRUE(window->GetData().IsNull() == false);
+    EXPECT_TRUE(CompareBlobs(*window, *parent));
+    EXPECT_TRUE(window->GetData() == parent->GetData());
```

Comment:
Add test to check `window.SetPosition( int )` work, because there is no that one